src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -397,12 +397,12 @@ private void CheckUserDefinedConversionSignature(DiagnosticBag diagnostics)
 
                 HashSet<DiagnosticInfo> useSiteDiagnostics = null;
 
-                if (same.IsDerivedFrom(different, ignoreDynamic: false, useSiteDiagnostics: ref useSiteDiagnostics)) // tomat: ignoreDynamic should be true, but we don't want to introduce breaking change. See bug 605326.
+                if (same.IsDerivedFrom(different, ignoreDynamicAndTupleNames: false, useSiteDiagnostics: ref useSiteDiagnostics)) // tomat: ignoreDynamic should be true, but we don't want to introduce breaking change. See bug 605326.
```

Comment:
We should be ignoring tuple names here. Feel free to enter a bug to track this.
Should be ignoring dynamic as well, but can't change that for compat reasons.